internal/bminventory/inventory.go

```diff
@@ -2789,27 +2789,34 @@ func (b *bareMetalInventory) getOLMOperators(cluster *common.Cluster, newOperato
 			return nil, common.NewApiError(http.StatusBadRequest, err)
 		}
 
+		operator.Properties = newOperator.Properties
+		monitoredOperators = append(monitoredOperators, operator)
+	}
+
+	operatorDependencies, err := b.operatorManagerApi.ResolveDependencies(cluster, monitoredOperators)
```

Comment:
operatorDependencies has monitored operators that were asked for + all their dependencies? 
in case of cnv it will be cnv + lvm?

Reply:
Yes, if CNV were selected it will store both CNV and LVM.
We cannot to do it at the beginning of that method, we need to create the monitored operators first.
In the future we need to do something with that function but this is different scope